templates/course/lesson-preview.php

```diff
@@ -63,6 +63,31 @@
 				do_action( 'llms_lesson_preview_before_title', $lesson )
 				?>
 				<div class="llms-lesson-title"><?php echo esc_html( get_the_title( $lesson->get( 'id' ) ) ); ?></div>
+
+				<?php
+					$has_quiz = $lesson->is_quiz_enabled();
+					if ( $has_quiz ) {
+						?>
+						<span class="llms-lesson-has-quiz">
+							<i class="fa fa-question-circle"></i>
+							<?php esc_html_e( 'Has Quiz', 'lifterlms' ); ?>
+						</span>
+						<?php
+					}
+				?>
+
+				<?php
+					// Does the lesson have an assignment?
```

Comment:
Could we remove the "what" comment since the `llms_lesson_has_assignment` function shows this already? I've been cleaning up any what-type comments especially since the "what" can change and the comment doesn't get updated down the road.

Reply:
Completed